### DISCUSSION TITLE
linux的demo 运行报错-pagSurface is nullptr!!!

### DISCUSSION THREAD

**Discussion by LiuGangR** [Tencent/libpag#1705] [Q&A] [answered]
GLDevice::Make() eglCreatePbufferSurface error=12296
----pagSurface is nullptr!!!

**Comment by kevingpqi123** [comment, marked as answer]
出现这个问题的原因是 OpenGL 相关依赖了两个动态库 libEGL.so、ibGLESv2.so，需要配置系统依赖
具体可以参考这篇文章：https://www.jianshu.com/p/0de77454f105
